### PR TITLE
Add Working examples to the xbacked-sdk

### PR DESCRIPTION
**Pull request by Jesulonimi21**
This Pr adds working examples to the xbacked-sdk

### CODE REVIEW

File: examples/feeCollection.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  const vault = new Vault({ id: 123 });
+  const vault = new Vault({id: 123});
 
   // distribute collateral fees to xUSD stakers and DAO
   // Receive 0.5% of the accured fees
-  await account.collectFees({ vault });
+  await account.collectFees({vault});
   // Settle accrued interest & distribute to DAO & Governance token stakers
-  await account.settleInterest({ vault });
+  await account.settleInterest({vault});
```
Comment (AwolDes) [inline examples/feeCollection.js:14]: Check the linting here

File: examples/getAllAccounts.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+const {getAllAccounts, Account, Vault} = require('..');
+const {loadStdlib} = require('@reach-sh/stdlib');
 
 (async () => {
   const account = new Account({
     mnemonic: 'lens sell....',
     network: 'LocalHost',
   });
   const VAULT_ID = 123;
-  const vault = new Vault({ id: VAULT_ID });
```
Comment (AwolDes) [inline examples/getAllAccounts.js:10]: Did the linter auto correct this? If it did, we should change it so it is the opposite styling.

**Review by AwolDes — APPROVED** [review]
Comment on code style - updates look good.

File: examples/getAllAccounts.js (hunk 2 above)
Comment (Jesulonimi21) [reply examples/getAllAccounts.js:10]: @AwolDes 
Please kindly review
Thanks

**Review by AwolDes — APPROVED** [review]

**Pull request merged**